Gems/Atom/RHI/Vulkan/Code/Source/RHI/PipelineState.cpp

```diff
@@ -27,16 +27,28 @@ namespace AZ
 
         PipelineLayout* PipelineState::GetPipelineLayout() const
         {
+            if (m_pipeline == nullptr)
+            {
+                return nullptr;
+            }
             return m_pipeline->GetPipelineLayout();
         }
 
         Pipeline* PipelineState::GetPipeline() const
         {
+            if (m_pipeline == nullptr)
+            {
+                return nullptr;
+            }
             return m_pipeline.get();
```

Comment:
Is this one necessary? it should just return the contained nullptr?

Reply:
good point, will restore